src/components/OpenSource/Talks.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import styled from 'styled-components'
+import breakpoint from 'styled-components-breakpoint'
+import remcalc from 'remcalc'
+
+import BlueBackground from '../Common/BlueBackground'
+import CompactVideoLink from '../Common/CompactVideoLink'
+import StyledLink from '../Common/StyledLink'
+import { Grid, Row, Col } from '../grid'
+import { SectionTitle } from '../Typography'
+
+const StyledImage = styled.img`
+  padding: ${remcalc(83)} 0 ${remcalc(15)};
+`
+
+const StyledSectionTitle = styled(Col)`
+  padding: 0 0 ${remcalc(35)};
+
+  ${breakpoint('smallTablet')`
+    padding-left: ${remcalc(21)};
+  `}
+
+  ${breakpoint('tablet')`
+    padding-left: ${remcalc(24)};
+  `}
+`
+
+const StyledTalksRow = styled(Row)`
+  padding-bottom: ${remcalc(35)};
+`
+
+const StyledTalksLink = styled(StyledLink)`
+  padding-bottom: ${remcalc(90)};
+`
+
+const TalkCol = ({
+  title,
+  type,
+  link,
+  cta,
+  featured,
+  additionalInfo,
+  colorCode
+}) => (
+  <CompactVideoLink
```

Comment:
What controls the outline colour?

Zeplin
![image](https://user-images.githubusercontent.com/15656538/58720186-66ed3e80-83c9-11e9-819c-c8c7983530f8.png)

Your branch
![image](https://user-images.githubusercontent.com/15656538/58720199-6d7bb600-83c9-11e9-8a2d-6dc3c2bc9d36.png)

Reply:
const MAX_VIDEOS is a good idea. Used already existing code which is pretty much doing what I was doing. Still need to resolve the outline color.

Reply:
On the Join Us page, the outline is already grey:
![image](https://user-images.githubusercontent.com/15656538/58808346-c047ae80-8611-11e9-9028-8179e908c768.png)

Perhaps look into how that was achieved?

Reply:
Hmm

Ok...judging by [Zeplin designs for `StandaloneVideoLink`](https://app.zeplin.io/project/5b4e666f0d4230a16dffd93a/screen/5c3e0570797c210e8b18b0d3):
![image](https://user-images.githubusercontent.com/15656538/58808907-d7d36700-8612-11e9-8c02-b88322d723d5.png)

the border, in line 77 of this code should be changed based on `themeVariation`:
https://github.com/yldio/yld.io/blob/5f40328f15a59d285b02363cae2b9668a92132bc/src/components/Common/BaseVideoLink.js#L72-L78

Reply:
ok, so it's something that I can fix but it will fix for every video using the dark themevariation.

Reply:
Done. I updated StandaloneVideoLink after I spoke with James.